postprocessing/pp-opus.c

```diff
@@ -128,176 +129,29 @@ int janus_pp_opus_process(FILE *file, janus_pp_frame_packet *list, int *working)
 			last_seq = tmp->seq;
 			steps++;
 		}
-		ogg_packet *op = op_from_pkt((const unsigned char *)buffer, bytes);
-		pos = (tmp->ts - list->ts) / 48 / 20 + 1;
 		JANUS_LOG(LOG_VERB, "pos: %06"SCNu64", writing %d bytes out of %d (seq=%"SCNu16", step=%"SCNu16", ts=%"SCNu64", time=%"SCNu64"s)\n",
 			pos, bytes, tmp->len, tmp->seq, diff, tmp->ts, (tmp->ts-list->ts)/48000);
-		op->granulepos = 960*(pos); /* FIXME: get this from the toc byte */
-		ogg_stream_packetin(stream, op);
-		g_free(op);
-		ogg_write();
-		ogg_flush();
+		av_packet_unref(pkt);
+		pkt->stream_index = 0;
+		pkt->data = buffer;
+                pkt->size = bytes;
```

Comment:
Broken indentation (spaces).

Reply:
Looks like in one of the machines I tested I do not have editorconfig support. Do you happen to have a clang-format setup I can use?